Shared/ThresholdKeyView.swift

```diff
@@ -28,6 +32,20 @@ struct ThresholdKeyView: View {
         tkeyReconstructed = false
         resetAccount = true
     }
+    
+    func changeQnA() async {
+        isLoadingChangePassword = true
+        let question = "what's your password?"
+        let answer = password
+        _ = try! await SecurityQuestionModule.change_question_and_answer(threshold_key: threshold_key, questions: question, answer: answer)
+        let key_details = try! threshold_key.get_key_details()
+        totalShares = Int(key_details.total_shares)
+        threshold = Int(key_details.threshold)
```

Comment:
If the try! fails the app will crash, rather wrap this in a `do ... catch` block,   see also `guard let`. Otherwise consider having this function defined with  `throws` and use the previous statement when it is initially called.
```suggestion
        _ = try await SecurityQuestionModule.change_question_and_answer(threshold_key: threshold_key, questions: question, answer: answer)
        let key_details = try threshold_key.get_key_details()
        totalShares = Int(key_details.total_shares)
        threshold = Int(key_details.threshold)
```

Reply:
done